docs/introduction/release_process.md

```diff
@@ -47,3 +47,13 @@ Two weeks before each feature release, we implement a code freeze on the master
 For patch releases, code freeze occurs the Thursday before the release. Patch releases also go through exporter tests and nightly CI tests.
 
 After all tests return no errors, we release the latest updates. You can find the most recent release in the `mbed-os` repository with the `latest` tag. A release note accompanies each release. The release notes for major and feature releases are longer and give an overview of the new features. The release notes for the patch releases include only a list of changes and known issues, if applicable. You can find our release notes on [the releases page](https://os.mbed.com/releases/) and on [the blog](https://os.mbed.com/blog/).
+
+## The API life cycle
+
+Mbed OS API are managed in three phases:
+
+1.	Experimental: No changes are considered breaking, so they can be changed in any release type. They are on the Mbed OS Master branch and can be included in any Mbed OS build, but are identified as experimental and users have to explicitly include them.
+1.	Stable: Can only be changed in a non-breaking manner. These are included by default in the Mbed OS full profile and, where appropriate, the bare metal profile.
+1.	Deprecated: API deprecation is a breaking change, and done only in major releases.
```

Comment:
I think this is a common misconcaption:
* Deprecated - marked to removal, but still available to the users (will generate a warning)
* Removal - deprecated API removed from the code base, not longer available to the user.

So when the API is deprecated it will be available to the users until next major release when it will be removed. The marking as "deprecated" can happen in minor relase.

Reply:
So do we have four phases, or a clarification for the three?

Reply:
I think 3 phases: Experimental, Stable, Deprecated; after the last one the API is removed in next major release. Deprecated should say that API is marked by deletion and its usage will generate a warning, it'll be removed in next major release. Marking API as deprecated is not a breaking change as the code using this API will continue to work until the next major release.